selfdrive/car/toyota/carcontroller.py

```diff
@@ -107,7 +107,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
 
     # we can spam can to cancel the system even if we are using lat only control
     if (frame % 3 == 0 and CS.CP.openpilotLongitudinalControl) or (pcm_cancel_cmd and Ecu.fwdCamera in self.fake_ecus):
-      lead = lead or CS.out.vEgo < 12.    # at low speed we always assume the lead is present do ACC can be engaged
+      lead = lead or (CS.out.vEgo < 12. and CS.out.vEgo > 0.001)    # at low speed we always assume the lead is present so ACC can be engaged
```

Comment:
```suggestion
      lead = lead or (CS.out.vEgo < 12. and (not CS.out.standstill))    # at low speed we always assume the lead is present so ACC can be engaged
```

Reply:
The standstill field is probably better suited. Does that also work correctly?

Reply:
One sec, let me test this

Reply:
This doesn't work, op can still engage at zero speed

Reply:
Strange. The threshold for standstill is the same (0.001). Only thing is it uses raw wheel speed instead of the Kalman filered one.

Can you make it work with the vEgoRaw, but with a larger threshold? The Kalman filter can behave a little erratic at very low speed causing vEgo to not be correct.

Reply:
It might be because openpilot mistaking my bushes as a lead, which happened before. I can try again on road this morning

Reply:
OK this does work.